### ISSUE TITLE
Problems with Binance UP/DOWN tokens

### ISSUE THREAD

**Issue by Prossi79** [freqtrade/freqtrade#4277]
## Describe your environment

  * Operating system: Ubuntu 20.04
  * Python Version:  3.8.5
  * CCXT version:  1.40.99
  * Freqtrade Version: develop-3a834929

  
Note: All issues other than enhancement requests will be closed without further comment if the above template is deleted or not filled out.

## Describe the problem:

pair list is "loosing" UP / DOWN token

### Steps to reproduce:

following pairlist setup in the config:

1) all UP / DOWN token being white-listed

2) following filter

    "pairlists": [

        {"method": "StaticPairList"},        
        {"method": "AgeFilter", "min_days_listed": 10},        
        {"method": "PrecisionFilter"},
        {"method": "PriceFilter", "low_price_ratio": 0.01},
        {"method": "SpreadFilter", "max_spread_ratio": 0.0075},
        {"method": "ShuffleFilter", "seed": 42}
     ],

  
### Observed Results:

  * if the Bot is being started almost all pairs out if the white list are available:
```
Using whitelist ['StaticPairList', 'AgeFilter', 'PrecisionFilter', 'PriceFilter', 'SpreadFilter', 'ShuffleFilter'] with 28 pairs
XTZDOWN/USDT, SXPUP/USDT, TRXDOWN/USDT, DOTUP/USDT, AAVEUP/USDT, LINKDOWN/USDT, EOSUP/USDT, XRPUP/USDT, FILUP/USDT, XTZUP/USDT, DOTDOWN/USDT, BNBDOWN/USDT, YFIUP/USDT, ETHUP/USDT, LTCUP/USDT, FILDOWN/USDT, BTCUP/USDT, BTCDOWN/USDT, SUSHIUP/USDT, ETHDOWN/USDT, XRPDOWN/USDT, UNIUP/USDT, BNBUP/USDT, ADAUP/USDT, TRXUP/USDT, LINKUP/USDT, EOSDOWN/USDT, LTCDOWN/USDT
```

  * after a while the Bot is "loosing" almost all pairs until he has only a few left, this behaviour I have not seen so far. I am running the same bot with "standard" USDT pairs, no issues here

```
Using whitelist ['StaticPairList', 'AgeFilter', 'PrecisionFilter', 'PriceFilter', 'SpreadFilter', 'ShuffleFilter'] with 6 pairs
LINKUP/USDT, SXPUP/USDT, BTCDOWN/USDT, ETHUP/USDT, XRPUP/USDT, YFIUP/USDT

```

### Relevant code exceptions or logs

the logs are looking good, nothing suspicious

  ```
2021-01-29 14:00:00,791 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:00:16,644 - freqtrade.data.converter - INFO - Missing data fillup for XRPUP/USDT: before: 999 - after: 1003
2021-01-29 14:00:16,665 - freqtrade.data.converter - INFO - Missing data fillup for BTCDOWN/USDT: before: 999 - after: 1003
2021-01-29 14:00:16,685 - freqtrade.data.converter - INFO - Missing data fillup for LTCUP/USDT: before: 999 - after: 1003
2021-01-29 14:00:16,705 - freqtrade.data.converter - INFO - Missing data fillup for YFIUP/USDT: before: 999 - after: 1003
2021-01-29 14:00:16,766 - freqtrade.data.converter - INFO - Missing data fillup for LINKUP/USDT: before: 999 - after: 1003
2021-01-29 14:00:16,809 - freqtrade.data.converter - INFO - Missing data fillup for ETHUP/USDT: before: 999 - after: 1003
2021-01-29 14:00:16,935 - freqtrade.data.converter - INFO - Missing data fillup for EOSUP/USDT: before: 999 - after: 1003
2021-01-29 14:00:16,976 - freqtrade.data.converter - INFO - Missing data fillup for SXPUP/USDT: before: 999 - after: 1003
2021-01-29 14:00:16,998 - freqtrade.data.converter - INFO - Missing data fillup for ETHDOWN/USDT: before: 999 - after: 1003
2021-01-29 14:00:17,019 - freqtrade.data.converter - INFO - Missing data fillup for DOTUP/USDT: before: 999 - after: 1003
2021-01-29 14:00:17,039 - freqtrade.data.converter - INFO - Missing data fillup for EOSDOWN/USDT: before: 999 - after: 1003
2021-01-29 14:00:21,074 - freqtrade.data.converter - INFO - Missing data fillup for XRPUP/USDT: before: 999 - after: 1003
2021-01-29 14:01:04,752 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:02:04,779 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:03:04,806 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:04:04,836 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:05:04,864 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:06:04,890 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:07:04,918 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:08:04,945 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:09:04,972 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:10:04,998 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:11:05,026 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:12:05,055 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:13:05,085 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:14:05,115 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'
2021-01-29 14:15:05,139 - freqtrade.worker - INFO - Bot heartbeat. PID=792, version='develop-3a834929', state='RUNNING'

  ```

**Comment by zn3x** [comment]
That's what `filter` is used for, I don't knew what do you expect. But when a pair is filtered, it is removed from the whitelist.
I think you need debug logging level if you want to see log of when pairs are removed

**Comment by xmatthias** [comment]
i don't think with this section of the log we'll find anything.

Debug-log should not be required - however, whenever a filter removes a pair, this is logged (so it's clear which filter removes the pair). That's not included, so further investigation will not be possible.

**Comment by Prossi79** [comment]
@mobrine1  ... yes and no - yes, it should filter but it should not remove all pairs step by step ... normally this kind of bot and pairlist config removes in average 10-20% of the whitelisted pairs (mainly due to spread filter), but usually spread goes back below 0,75% and it would put the pair back to the white list, thus over time the whitelisted pairs are constantly fluctuating around one average value and not going down to zero.

@xmatthias ... pls find enclosed the whole log file for further investigations. btw after re-starting the bot he got a full pairlist (apart from the ones being filtered out immediately after start-up)
[freqtrade-service-5.zip](https://github.com/freqtrade/freqtrade/files/5897664/freqtrade-service-5.zip)

**Comment by xmatthias** [comment]
Seems like the pairlist never resets when using Staticpairlist - so once the pair is removed, it'll never reappear.

Will be fixed in #4296 - which i'll merge after ci passes.
